server/src/main/java/org/elasticsearch/action/synonyms/PutSynonymsAction.java

```diff
@@ -40,31 +45,40 @@ public PutSynonymsAction() {
     }
 
     public static class Request extends ActionRequest {
-        private final String synonymssetId;
-        private final SynonymsSet synonymsset;
```

Comment:
Removed `SynonymSet` class according to the TODO item in the previous PR. Parsing is thus included in the PUT request itself - we could move the parsing to the SynonymRule class as well.